kinto/tests/test_views_collections_schema.py

```diff
@@ -212,3 +212,12 @@ def test_records_are_validated_on_patch(self):
         self.app.patch_json(record_url,
                             {'data': record},
                             headers=self.headers)
+
+    def test_additional_properties_are_rejected(self):
+        record_id = '5443d83f-852a-481a-8e9d-5aa804b05b08'
+        record = VALID_RECORD.copy()
+        record['extra'] = 'blah!'
+        self.app.put_json('%s/%s' % (RECORDS_URL, record_id),
+                          {'data': record},
+                          headers=self.headers,
+                          status=400)
```

Comment:
I would also validate that the error message give enough feedback for the user to solve the issue.